test/cqlpy/test_materialized_view.py

```diff
@@ -1538,3 +1538,12 @@ def test_alter_table_add_select_star(cql, test_keyspace):
             cql.execute(f'INSERT INTO {base} (p,a,b,c) VALUES (0,1,2,3)')
             assert {(0,1,2,3),(1,2,3,None)} == set(cql.execute(f"SELECT p,a,b,c FROM {base}"))
             assert {(0,1,2,3),(1,2,3,None)} == set(cql.execute(f"SELECT p,a,b,c FROM {mv}"))
+
+# Test view representation in system.* tables
+def test_view_in_system_tables(cql, test_keyspace):
+    with new_test_table(cql, test_keyspace, "p int PRIMARY KEY, v int") as base:
+        with new_materialized_view(cql, base, '*', 'v,p', 'v is not null and p is not null') as view:
+            res = [ f'{r.keyspace_name}.{r.view_name}' for r in cql.execute('select * from system.built_views')]
+            assert view in res
+            res = [ f'{r.table_name}::{r.index_name}' for r in cql.execute('select * from system."IndexInfo"')]
+            assert view.split('.')[1] not in res
```

Comment:
The end of this test doesn't seem right... Clearly if you put in res a list of strings like name::name, with those double colons, the view's name will not be in it, so what's the point of testing it?
I would just drop the last two lines from this test.

Reply:
Dropping no, the original boost test checks that MVs are _not_ reported in this table, so it should be here as well. But you're right, the wrong line is checked against the result. Will fix